terraform/code.gov.tf

```diff
@@ -0,0 +1,33 @@
+variable "env_name" {}
+
+resource "aws_route53_zone" "code_toplevel" {
+   name = "code.gov"
+
+   tags {
+     Project = "dns"
+   }
+}
+
+resource "aws_route53_record" "www" {
+  zone_id = "${aws_route53_zone.code_toplevel.zone_id}"
+  name = "code.gov."
+  type = "A"
+
+  alias {
+    name = "s3-website-us-east-1.amazonaws.com."
+    zone_id = "XXX"
```

Comment:
@jezhumble can you help me figure out what to do here?

Unsure how to populate `zone_id` and how to specify the alias target (which I believe should be the ELB that @dlapiduz setup for us).

Thanks!!